webroot/adm/keysets.html

```diff
@@ -17,6 +17,8 @@ <h3>Inputs</h3>
 <input type="text" id="siteId" name="siteId">
 <label for="keysetId">Keyset Id:</label>
 <input type="text" id="keysetId" name="keysetId">
+<label for="keysetName">Set Keyset Name:</label>
+<input type="text" id="keysetName" name="keysetName">
 <label for="addSiteIds">Add Site Ids:</label>
```

Comment:
I still think _Add/Remove Site Ids_ should be renamed to something more descriptive like 
_Add Site Ids to Allowed Sites_ and _Remove Site Ids from Allowed Sites_ 

to be more descriptive as Add Site Ids could be confusing to the _Site Id_ field

Reply:
I like the idea of "Set" but I'm worried about people blowing away a large list accidentally. How would that be solved?